app/controllers/orders_controller.rb

```diff
@@ -4,7 +4,14 @@ class OrdersController < ApplicationController
   before_action :authenticate_employee!
 
   def index
-    @orders = Order.where(employee: current_employee)
+    @orders = case params[:filter]
```

Comment:
this is okay, but I like to create a query object for this. A neat and quick gem to set up is called `searchlight`.

Reply:
used searchlight for this, thanks for tip :)